.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
The package-lock is alread in the gitignore

https://github.com/fastify/fastify-compress/blob/ee2cd0605dd0fb1b49b5207e1c617efe31162f68/.gitignore#L139

Why is this necessary?

Reply:
Here the discussion already took place